corona-warn-app/cwa-website
# Fix blog archive link to bug-fixing image (resolves #1790)

This PR fixes a bad link to https://www.coronawarn.app/assets/img/news/bug-fixing.jpg in
- https://www.coronawarn.app/en/blog/archive/ "News Archive" and
- https://www.coronawarn.app/de/blog/archiv/ "News Archiv"

described in issue https://github.com/corona-warn-app/cwa-website/issues/1790. The issue only affects older browsers which are not HTML5 compatible.

It changes the image link to point to the available image https://www.coronawarn.app/assets/img/news/bug-fixing.jpg.

## Steps to verify

Open 
en/blog/archive/ "News Archive" and
de/blog/archiv/ "News Archiv"

in Internet Explorer either on a local test http://localhost:8000 or in production on https://www.coronawarn.app and check that the graphic displays as shown below:

![bug-fixing restored](https://user-images.githubusercontent.com/66998419/135624510-badc9d9b-604d-4f60-94ef-0a4815791e48.jpg)

Now @tklingbeil can be seen again! 🙂
